stable/jenkins/templates/config.yaml

```diff
@@ -243,6 +243,9 @@ data:
 {{- end }}
 {{- if .Values.Master.InitScripts }}
     mkdir -p /var/jenkins_home/init.groovy.d/;
+    {{- if .Values.Master.OverwriteConfig }}
+      rm -f /var/jenkins_home/init.groovy.d/*.groovy
```

Comment:
The indetation here seems wrong

Reply:
this is still over indented (by 2)

Reply:
"think" i fixed it here. At least it looks aok with `helm template`, but lemme know 👍